supported-standards/lens-metadata-standards/publication/livestream/1.0.0/README.md

```diff
@@ -0,0 +1,151 @@
+   */
+  lens: LivestreamMetadataDetails;
+
+  /**
+   * If you use an appId and you only want your app to be able to publish under it,
+   * you will need to sign the `lens` object with your server private key
+   * and populate the signature here. We will need your public key to verify this.
+   * If the signature does not match what we have in our mappings it will not be surfaced
+   * in the API.
+   */
+  signture: string;
+}
+
+interface LivestreamMetadataDetails extends MetadataCommon {
+  /**
+   * The livestream title
+   */
+  title?: string;
+
+  /**
+   * unix timestamp (unix timestamp)
+   */
+  startsAt: number;
+
+  /**
+   * unix timestamp (unix timestamp)
+   */
+  endsAt?: number;
+
+  /**
+   * Some livestream platforms have the playback url as a separate url
+   * If you do not liveUrl and playbackUrl should just be the same to avoid
+   * confusion with optional fields
+   */
+  playbackUrl: Url;
+
+  /**
+   * Some livestream platforms have the live url as a separate url
+   */
+  liveUrl: Url;
+
+  /**
+   * The data can not be changed so you can put in an API endpoint to know
+   * if it is still live or not for clients to be able to check
+   */
+  isLiveNowApiCall?: Url;
+
+  /**
+   * Main content focus that for this publication
+   */
+  mainContentFocus: PublicationMainFocus.LIVESTREAM;
+
+  /**
+   * The other attachments you want to include with it
+   */
+  attachments?: (MediaImage | MediaVideo | MediaAudio)[];
+}
+```
+
+## Field extra requirements
+
+`isLiveNowApiCall` response should be:
+
+```json
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "isLive": {
+      "type": "boolean"
+    }
+  },
+  "required": ["isLive"]
+}
+```
+
+`playbackUrl` and `liveUrl` must have the `lensId` in the query string parameter example:
+
+`https://decentralized-media-server.com/stream.m3u8?lensId=0x2c6b-0x10`
+
+## JSON example
+
+Will use all the fields but remember a lot are optional you can use the JSON schema to see this.
+
+```json
+{
+  "$schema": "https://raw.githubusercontent.com/lens-protocol/LIPs/feat/metadata-standards/lens-metadata-standards/publication/livestream/1.0.0/schema.json",
+  "name": "My livestream",
+  "description": "My livestream",
+  "attributes": [],
+  "image": "https://myimage.com/image.png",
+  "external_url": "https://livestream.com",
+  "animation_url": "https://playbackurl.com/trailer.mp4",
+  "lens": {
+    "id": "1234",
+    "content": "Come to my livestream tonight",
+    "locale": "en-US",
+    "mainContentFocus": "LIVESTREAM",
+    "title": "My livestream",
+    "startsAt": "2023-05-24T12:46:10.987Z",
+    "endsAt": "2023-05-24T19:46:10.987Z",
+    "playbackUrl": "https://playbackurl.com/playback.m3u8?lensId=0x2c6b-0x10",
+    "liveUrl": "https://liveurl.com/live.m3u8?lensId=0x2c6b-0x10",
+    "isLiveNowApiCall": "https://api.com/is-live",
+    "tags": ["livestream"],
+    "attachments": [
+      {
+        "type": "PNG",
```

Comment:
attachment type can be formatted as contentType/mediaType. !?

- image/png
- audio/mp3
- video/mpeg